lib/lte_link_control/lte_lc.c

```diff
@@ -690,9 +698,12 @@ static int connect_lte(bool blocking)
 			LOG_INF("Network connection attempt timed out");
 
 			if (IS_ENABLED(CONFIG_LTE_NETWORK_USE_FALLBACK) &&
-			    (sys_mode_target == sys_mode_preferred)) {
-				sys_mode_target = sys_mode_fallback;
-				retry = true;
+				(tries > 0) ) {
```

Comment:
Indentation here should be level with `IS_ENABLED(CONFIG_LTE_NETWORK_USE_FALLBACK)`

Reply:
Corrected!